src/components/molecules/Footer/Footer.tsx

```diff
@@ -5,12 +5,15 @@ import {
   PRIVACY_POLICY,
   TERMS_AND_CONDITIONS_URL,
 } from "settings";
+import { getExtraLinkProps } from "utils/url";
 
 import "./Footer.scss";
 
 export const Footer = () => (
   <div className="footer-wrapper">
-    <a href={TERMS_AND_CONDITIONS_URL}>Terms of Use</a>
+    <a href={TERMS_AND_CONDITIONS_URL} {...getExtraLinkProps(true)}>
```

Comment:
[nit] the `true` in getExtraLinkProps doesn't tell us a lot. We should refactor `getExtraLinkProps` to take a `{external: true}` style 'props' param.